templates/taxi/driver_list.html

```diff
@@ -7,7 +7,11 @@ <h1>Driver list</h1>
       {% for driver in driver_list%}
         <li>
           <a href="{% url "taxi:driver_detail" pk=driver.id %}">{{ driver.username }}</a>
-          ({{ driver.first_name }} {{ driver.last_name }})
+            {% if driver == user %}
+                (Me)
```

Comment:
You should display first_name and last_name for yourself too.

Reply:
done